README.md

```diff
@@ -47,4 +47,15 @@ binding, [for example](https://github.com/jhaber/dropwizard-guicier-example/blob
 There is an [example project](https://github.com/jhaber/dropwizard-guicier-example) you can clone and play with if you'd like to get
 going right away. 
 
+## Upgrading from dropwizard-guice
+There are a couple important changes to be aware of when upgrading from [dropwizard-guice](https://github.com/HubSpot/dropwizard-guice).
+
+### AutoConfig has been removed
+[Reasoning and potential workarounds here](https://github.com/HubSpot/dropwizard-guicier/issues/41). The workarounds are however, not recommended.
+
+### Explicit Bindings Required
+All objects that you wish to inject must be explicitly bound. Just-in-time bindings are not supported by default. This is the case despite you enabling explicit binding in your own module.
```

Comment:
> This is the case despite you enabling explicit binding in your own module.

Not sure I follow, can you clarify this?

Reply:
The behavior that I saw was because the `GuiceEnforcer` is enabled by default, I didn't have to enable explicit bindings in my own project.

Reply:
Ah I see, did you mean: "This is the case despite you **not** enabling explicit binding in your own module."

Reply:
Yup! Sorry, I'll fix that. Wrote these docs in a hurry.